fusor-ember-cli/app/mixins/openshift-mixin.js

```diff
@@ -95,182 +95,252 @@ export default Ember.Mixin.create({
     }
   ),
 
-  vcpuNeeded: Ember.computed('numMasterNodes', 'numWorkerNodes', 'masterVcpu', 'nodeVcpu', function() {
+  vcpuNeeded: Ember.computed('numMasterNodes', 'numWorkerNodes', 'masterVcpu', 'workerVcpu', function() {
     if ((this.get('numMasterNodes') > 0) && (this.get('masterVcpu') > 0) &&
-        (this.get('numWorkerNodes') >= 0) && (this.get('nodeVcpu') > 0) ) {
+        (this.get('numWorkerNodes') >= 0) && (this.get('workerVcpu') > 0) ) {
       return ((this.get('numMasterNodes') * this.get('masterVcpu')) +
-              (this.get('numWorkerNodes') * this.get('nodeVcpu')));
+              (this.get('numWorkerNodes') * this.get('workerVcpu')));
     } else {
       return 0;
     }
   }),
 
-  ramNeeded: Ember.computed('numMasterNodes', 'numWorkerNodes', 'masterRam', 'nodeRam', function() {
+  ramNeeded: Ember.computed('numMasterNodes', 'numWorkerNodes', 'masterRam', 'workerRam', function() {
     if ((this.get('numMasterNodes') > 0) && (this.get('masterRam') > 0) &&
-        (this.get('numWorkerNodes') >= 0) && (this.get('nodeRam') > 0) ) {
+        (this.get('numWorkerNodes') >= 0) && (this.get('workerRam') > 0) ) {
       return ((this.get('numMasterNodes') * this.get('masterRam')) +
-               (this.get('numWorkerNodes') * this.get('nodeRam')));
+               (this.get('numWorkerNodes') * this.get('workerRam')));
     } else {
       return 0;
     }
   }),
 
-  diskNeeded: Ember.computed('numMasterNodes', 'numWorkerNodes', 'masterDisk', 'nodeDisk', 'storageSize', function() {
+  diskNeeded: Ember.computed('numMasterNodes', 'numWorkerNodes', 'masterDisk', 'workerDisk', 'storageSize', function() {
     if ((this.get('numMasterNodes') > 0) && (this.get('masterDisk') > 0) &&
-        (this.get('numWorkerNodes') >= 0) && (this.get('nodeDisk') > 0) && (this.get('storageSize') > 0)) {
+        (this.get('numWorkerNodes') >= 0) && (this.get('workerDisk') > 0) && (this.get('storageSize') > 0)) {
       return ((this.get('numMasterNodes') * this.get('masterDisk')) +
-              (this.get('numWorkerNodes') * this.get('nodeDisk')) +
+              (this.get('numWorkerNodes') * this.get('workerDisk')) +
               (this.get('numWorkerNodes') * this.get('storageSize')));
     } else {
       return 0;
     }
   }),
 
-  vcpu1Needed: Ember.computed('masterVcpu', function() {
-    return parseInt(this.get('masterVcpu'));
+  vcpu2MasterNeeded: Ember.computed('numWorkerNodes', 'masterVcpu', 'workerVcpu', function() {
```

Comment:
There's a lot of repetition in this module; can we consolidate these enumerated properties somehow to cut down on this?